sklearn/utils/tests/test_validation.py

```diff
@@ -1172,3 +1173,8 @@ def test_check_sparse_pandas_sp_format(sp_format):
     assert sp.issparse(result)
     assert result.format == sp_format
     assert_allclose_dense_sparse(sp_mat, result)
+
+
+def test_sparse_matrix_to_numpy():
+    y = sp.csr_matrix((13, 1))
```

Comment:
you could add a comment like `# check if column_or_1d accepts sparse input PR#16800`